clients/python/README.md

```diff
@@ -105,12 +104,12 @@ Here some examples of **BAD** bad choices for `source_id`:
 ### Advanced configuration
 
 Samsara SDK buffers events and peridically flushes the events to the
-Samsara API. A ring buffer is used for this purpose. The events are
+Samsara API. `collections.deque` is used for this purpose. The events are
```

Comment:
the thing is a deque is not a ring buffer so I'd rather not say a ring buffer is being used if it (or any analogous structure) isn't.